### DISCUSSION TITLE
New Ikea Tradfri bulb unsupported LED1934G3_E27 250lm

### DISCUSSION THREAD

**Discussion by radicalcurtain** [Koenkk/zigbee2mqtt#10944] [Devices]
Hello,

I have plenty of other IKEA devices and they work well. But TRADFRIbulbE27WWclear250lm does not.

In the devices list there is an IKEA Trådfri bulb that seemingly _should_ match but doesn't. I notice however that the lamp looks slightly different from the picture on zigbee2mqtt.
![Trådfri bulb not found](https://user-images.githubusercontent.com/18324507/150632511-19871d71-dd8a-4695-8659-f152b4beed4e.png)
![trådfri led](https://user-images.githubusercontent.com/18324507/150632732-fd427d0d-101a-448e-8a1a-6b4c1ff64879.jpg)

**Comment by Koenkk** [comment]
`TRADFRIbulbE27WWclear250lm` should be supported in the latest dev branch (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) https://github.com/Koenkk/zigbee-herdsman-converters/blob/ae924d92ad6c8737c34dd35302b3be888454dd93/devices/ikea.js#L345

**Reply by radicalcurtain** [reply]
Thanks for the quick reply! If I'm too lazy to switch to the dev branch - any estimates when it will hit the "regular" version of zigbee2mqtt?

**Reply by Koenkk** [reply]
next release is on 1 February

**Reply by radicalcurtain** [reply]
I shall endure. Many thanks!

**Reply by radicalcurtain** [reply]
Hello, I notice it's the 5th of February and the release in the repository is 1.23 - however over in HomeAssistant it's still 1.18 even though the auto-updater is enabled. Is there some additional lag in the versions somehow?

**Reply by Koenkk** [reply]
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0
